ros_cross_compile/dependencies.py

```diff
@@ -95,3 +97,22 @@ def assert_install_rosdep_script_exists(
             'Rosdep installation script has never been created, you need to run this without '
             'skipping rosdep collection at least once.')
     return True
+
+
+class DependenciesStage(PipelineStage):
+    """Wrapper class for running the dependencies stage of the pipeline."""
```

Comment:
This comment is not very informative if we pick it apart:
* "Wrapper class" - the fact that it's a `class` already tells us this
* "dependencies stage of the pipeline" - this is nearly verbatim the declaration `DependenciesStage(PipelineStage)`

So essentially we have added 0 bits of information by having this docstring. A better one would actually describe what any of that _means_

```
"""This stage of the cross-compilation pipeline evaluates the workspace using rosdep to determine what external dependencies are needed for building.

It outputs a script into the internals directory that will install those dependencies for the target platform.
```